Doc/library/re.rst

```diff
@@ -965,12 +965,12 @@ form.
    Clear the regular expression cache.
 
 
-.. exception:: error(msg, pattern=None, pos=None)
+.. exception:: ReCompileError(msg, pattern=None, pos=None)
```

Comment:
We need an explicit index reference for `error`.

Reply:
how would I add and explicit index reference ? `.. error:` on top ?